app/views/articles/new.html.erb

```diff
@@ -1,29 +1,35 @@
-<h1>New Article</h1>
+<tr class="header-space"></tr>
+<tr>
+  <td>
+    <h1>New Article</h1>
 
-<%= form_with model: @article do |form| %>
-  <div>
-    <div>
-      <%= form.label :title %>
-    </div>
-    <%= form.text_field :title %>
-    <% @article.errors.full_messages_for(:title).each do |message| %>
-      <div><%= message %></div>
-    <% end %>
-  </div>
+    <%= form_with model: @article do |form| %>
```

Comment:
Since you made `article` a helper method, I would use it here instead of `@article`.